File: Lista pt3/Exercicio03Arrays.java
public class Exercicio03Arrays {
    public static void main(String[] args) {

        int[] impares = new int[100];

        int posicao = 0;

        for (int i = 1; i <= 200; i += 2) {

            impares[posicao] = i;

            posicao++;
        }

        for (int numero : impares) {
            System.out.println(numero);
        }
    }
}

File: Lista pt3/Exercicio03Arrays.js
let impares = [];

for (let i = 0; i <= 200; i++) {

    if (i % 2 !== 0) {
        impares.push(i);
    }
}

console.log("Números ímpares:");

for (let i = 0; i < impares.length; i++) {
    console.log(impares[i]);
}